packages/client/src/views/GrantDetails.vue

```diff
@@ -434,11 +443,19 @@ export default {
     selectableOption(option) {
       return option.status_code !== HEADER;
     },
+    goBack() {
+      this.$router.go(-1);
```

Comment:
This is a bit of personal preference, so feel free to ignore if you prefer this setup, but I tend to prefer inlining really simple one-liners instead of adding the indirection of a method like this. So like `@click="$router.back()` instead of `@click="goBack"`.